modmesh/core.py

```diff
@@ -74,9 +74,7 @@
     'Bezier3dFp64',
     'WorldFp32',
     'WorldFp64',
-
-    # TODO: have a way to enable this according to `USE_PYTEST_HELPER_BINDING`
-    # 'testhelper',
+    'testhelper',
```

Comment:
if not build with `testhelper` wrapper, it contains only `PYTEST_HELPER_BINDING_BUILT`